experimental/devices/pca9548/pca9548.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package pca9548
+
+import (
+	"errors"
+	"strconv"
+	"sync"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/conn/physic"
+)
+
+// DefaultOpts is the recommended default options.
+var DefaultOpts = Opts{Address: 0x70}
+
+// Opts is the pca9548 configuration.
+type Opts struct {
+	// Address pca9548 I²C Address. Valid addresses for the NXP pca9548 are 0x70
+	// to 0x77. The address is set by pulling A0~A2 low or high. Please refer
+	// to the datasheet.
+	Address int
```

Comment:
I think we should standardize on `Addr` everywhere. I've realized there's also `I2CAddress` used, will update that independently.